backend/api/Services/InspectionFindingsService.cs

```diff
@@ -0,0 +1,21 @@
+﻿using Api.Database.Context;
+using Api.Database.Models;
+using Microsoft.EntityFrameworkCore;
+
+
+namespace Api.Services
+{
+    public class IInspectionFindingService()
+    {
+
+        public async Task<List<InspectionFinding>> RetrieveInspectionFindings(TimeSpan timeSpan, FlotillaDbContext context)
```

Comment:
Inject the context into the service like the other services, and remove it as an input to this function

Reply:
Fixed